DuckDuckGo/Subscription/ViewModel/SubscriptionSettingsViewModel.swift

```diff
@@ -271,7 +266,8 @@ final class SubscriptionSettingsViewModel: ObservableObject {
 
     @MainActor
     func showTermsOfService() {
-        self.openURL(SettingsSubscriptionView.ViewConstants.privacyPolicyURL)
+        let privacyPolicyQuickLinkURL = URL(string: AppDeepLinkSchemes.quickLink.appending(SettingsSubscriptionView.ViewConstants.privacyPolicyURL.absoluteString))!
+        openURL(privacyPolicyQuickLinkURL)
```

Comment:
Open the link in DDG app instead redirecting it to default browser (which may be different)